tutorials/standalone-tensorflow-raspberry-pi/index.md

```diff
@@ -0,0 +1,168 @@
+
+    ``` 
+    print '--- importing packages'
+
+	from tensorflow.contrib import predictor
+	import base64
+	import sys
+	import json
+	import subprocess
+	import datetime
+
+	“”” 
+	change in the line below the directory where you hosted your model
+	this is the directory where the .pb file and ‘variables’ directory
+	are hosted. Therefore make sure to add the ‘/1’ for the version of 
+	the model if necessary
+	“””
+	model_dir = '/tf_server/flowers_model/1'
+    ```
+
+
+## Running the Tensorflow Server
+Type the line below to start the Tensorflow server. Note that loading the Tensorflow libraries and Tensorflow saved model may take several minutes to load.
+
+    python /standalone-tensorflow-raspberry-pi/tf_server.py
+
+Once the model is loaded and you read “press enter to take picture”, press “Enter” to predict the type of flower. The lines of the response represent the probability that each of the flower types in the order below was photographed:
+
+daisy
+dandelion
+roses
+sunflowers
+tulips
+
+## Testing the solution
+You can test the solution by capturing the images below with the camera attached to the Raspberry Pi. The exact values of the prediction may vary depending on the quality of the image.
+
+| Flower type | Sample image | Expected result |
+| ----------- | ------------ | --------------- |
+| Daisy | [Link to daisy image](https://www.publicdomainpictures.net/pictures/40000/velka/daisy-flowers-white.jpg) | First line of the results with highest probability |
+| Dandelion | [Link to dandelion image](https://cdn.pixabay.com/photo/2013/07/25/16/21/dandelion-167112_960_720.jpg) | Second line of the results with highest probability |
+| Rose | [Link to rose image](http://storage.googleapis.com/cloud-ml-data/img/flower_photos/roses/921984328_a60076f070_m.jpg) | Third line of the results with highest probability |
+| Sunflower | [Link to sunflower image](https://cdn.pixabay.com/photo/2016/08/28/23/24/sunflower-1627193_960_720.jpg) | Fourth line of the results with highest probability |
+| Tulip | [Link to tulip image](https://www.publicdomainpictures.net/pictures/20000/velka/single-tulip-29612970075253BV.jpg) | Fifth line of the results with highest probability |
+
+## Extensions to this lab
+This lab can be extended to a more comprehensive solution that can learn fast by incorporating new input data from hundreds of devices and processing it by leveraging the power of the cloud. Other components of GCP can allow for analytics and team collaboration.
+
+In the architecture diagram below, we have incorporated IoT Core and Pub/Sub that allow to connect and manage hundreds of IoT devices and ingest millions of messages per second. In addition, Dataflow, BigQuery, Dataproc, and Data Studio enable building powerful analytics solutions based on managed services and collaborating with team members. 
```

Comment:
When referring to 'Cloud Pub/Sub', we need to include the 'Cloud' part. The same goes for some of the product names in the next senence. Since you're a Google employee, you can see the internal "Branding guidelines" page, which gives a table of product names with guidance like this.

Reply:
I made corrections to Cloud IoT Core, Cloud Pub/Sub, Cloud Dataflow, Google BigQuery, and Cloud Dataproc.